lightning/src/ln/channel.rs

```diff
@@ -773,7 +774,7 @@ impl<ChanSigner: ChannelKeys> Channel<ChanSigner> {
 			channel_transaction_parameters: ChannelTransactionParameters {
 				holder_pubkeys: pubkeys,
 				holder_selected_contest_delay: config.own_channel_config.our_to_self_delay,
-				is_outbound_from_holder: false,
+				is_outbound_from_holder: is_outbound,
```

Comment:
Note that in this context (`new_from_req`) we're handling an incoming `channel_open` message so it is always inbound. I think the outbound parameter can be dropped from `is_unsupported_shutdown_script` anyway, so should just drop the `is_outbound` variable.

Reply:
If the outbound parameter check is dropped from `is_unsupported_shutdown_script`, is it kept anywhere in the context of handling shutdown?